src/pydata_sphinx_theme/assets/styles/components/_breadcrumbs.scss

```diff
@@ -14,16 +14,12 @@ ul.bd-breadcrumbs {
     display: flex;
     align-items: center;
     white-space: nowrap;
-    flex-shrink: 0;
+    overflow-x: hidden;
 
-    &.active {
-      flex-shrink: 1;
+    a,
+    .ellipsis {
```

Comment:
Now the `text-overflow: ellipsis` rule applies to all the breadcrumbs not just the last/current one